### DISCUSSION TITLE
Help with contact forms and Woocommerce

### DISCUSSION THREAD

**Discussion by gurds91** [holger1411/understrap#1796] [General]
i am building my first site using Understrap, I took the Instant Understrap course recently.

Can you integrate Elementor Pro into Understrap for stuff like forms and WooCommerce? I tried to integrate elementor form but it will break stuff if you are using Gutenberg Word press editor for things. I like the fact that Bootstrap is built into Understrap. 

Failing Elementor, how can we integrate working forms and WooCommerce functionality into using Understrap, this wasn't covered in the course other than just saying that contact form 7 plugin is used.

Could you please guide me through how to integrate forms and WooCommerce into Understrap, is this free or cost associated with it. I have ok knowledge of php and know you can write code yourself for a working php form but it is cumbersome and Understrap apparently comes with a section for styling contact 7 forms. 

Please help,  I know there is an Understrap accelerator program which I tried to join but it says I need to contact you guys (message popped up)

**Comment by bacoords** [comment]
Understrap is not specifically built to integrate with Elementor, but it easily can. That being said, it is built to work with most common form builders (Contact Form 7, Ninja Forms, Gravity Forms, and WP Forms are some of the ones we use pretty frequently) and it is meant to integrate with WooCommerce.

**Comment by gurds91** [comment]
Is it possible to integrate it free the contact form 7 and WooCommerce or
are there associated costs

**Comment by bacoords** [comment]
If you're unclear on how to use plugins like Contact Form 7 or WooCommerce, I would recommend checking out some of the WordPress training material: https://learn.wordpress.org/?s=install+plugin

Both plugins are free but have additional premium options.

**Comment by gurds91** [comment]
Thanks Brian, I learnt a lot from you on the instant understrap course, you are very clever.

Is contact form 7  css styling there in understrap- whereabouts in css? Also if you copy and paste some files from parent theme to child them but  keep the same name, then you will be looking in the child theme but it won't affect anything in parent theme?
